portfolio/src/main/webapp/style.css

```diff
@@ -316,3 +315,62 @@ section .courses ul li span {
   margin: 5px;
 }
 
+/* -------------- Project Hover Cards --------------*/
+
+.container {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items:center;
+  flex-wrap: wrap;
+  padding: 40px 30px 30px 30px;
+}
+
+.card {
+  max-width: 400px;
+  height: 300px;
+  position: relative;
+  background: #fff;
+  margin: 30px 10px;
+  padding: 25px 15px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 5px 20px rgba(0,0,0,.3);
+  transition: 0.5s ease-in-out;
+  border-radius: 5px;
+}
+
+.card:hover {
+  height: 450px;
```

Comment:
It doesn't look like you took my advice about how to implement the hover effect :'((((